test/models/subscription_status_test.rb

```diff
@@ -17,30 +17,40 @@ def test_get_status
     end
 
     def test_to_status_valid
+      host.organization.stubs(:simple_content_access?).returns(false)
```

Comment:
optional: since most of these test cases use this same stub, it *could* go in a before method at the top. up to you

Reply:
Refactored to a `setup` method. Will start to look at the other test failures.